### PR TITLE
ChoiceGroup: Add description prop for options

### LINKED ISSUE #10920: Multiple lines/description in ChoiceGroup
#### Describe the feature that you would like added

It would be a really nice feature to have multiple lines on the radio group for use-cases where you need a description to the radio option. Personally I would prefer a solution where you can pass it with the options like this:

```
options: [
    {
      key: 'A',
      text: 'Ordinær registrering',
      description: 'Alle næringsdrivende som har omsetning som omfattes av merverdiavgiftsloven må registrere seg i Merverdiavgiftsregisteret.'
    },
    {
      key: 'B',
      text: 'Frivillig registrering',
      description: 'Enkelte virksomheter som i utgangspunktet ikke er mva-pliktig, kan søke om frivillig registrering. Eks. utleie av bygg og anlegg, bortforpaktere av landbrukseiendom, utbygger av vann- og avløpsanlegg, skogsveiforeninger, baneanlegg.'
    }
  ]
```

#### What component or utility would this be added to

ChoiceGroup

#### Additional context/screenshots
radioButtonGroup

### PR DESCRIPTION
**Pull request by malmil**
#### Pull request checklist

- [x] Addresses an existing feature request: Fixes #10920

#### Description of changes

Adds possibility to add description as a prop for options in ChoiceGroup

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10930)

### CODE REVIEW

**Comment by malmil** [comment]
Hey @ecraig12345 @srideshpande
I'm sorry for several reruns of checks. This is my first PR to fabric and I don't think I've fully understood the flow. Please correct me if I should have done something differently. Also, I'm I supposed to provide updates on snapshots?

**Comment by marygans** [comment]
> Hey @ecraig12345 @srideshpande
> I'm sorry for several reruns of checks. This is my first PR to fabric and I don't think I've fully understood the flow. Please correct me if I should have done something differently. Also, I'm I supposed to provide updates on snapshots?

You updated the snapshots correctly and it LGTM. @ecraig12345 or @srideshpande can take a look!

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts [hunk 1]
```diff
@@ -249,6 +249,10 @@ export const getStyles = (props: IChoiceGroupOptionStyleProps): IChoiceGroupOpti
         selectors: {
           '.ms-ChoiceFieldLabel': {
             display: 'inline-block'
+          },
+          '.ms-ChoiceFieldDescription': {
+            padding: '5px 0 0 26px',
+            color: semanticColors.disabledBodyText
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts]: don't use disabledBodyText for text that isn't actually disabled.

**Comment by micahgodbolt** [comment]
@ecraig12345 just beat me to it. unless this is a design driven change, i'd like to avoid adding props and no approved styles (like the disabled text).

This is currently possible to do with the current controls by using the onRenderLabel props like this:

https://codepen.io/micahgodbolt/pen/GRRvOxg?editors=1010

**Comment by ecraig12345** [comment]
Forgot to say, thank you @malmil for the submission--we really do appreciate your willingness to invest in Fabric! And sorry about not commenting about this concern sooner.

**Comment by malmil** [comment]
> @ecraig12345 just beat me to it. unless this is a design driven change, i'd like to avoid adding props and non approved styles (like the disabled text).
> 
> This is currently possible to do with the current controls by using the onRenderLabel props like this:
> 
> https://codepen.io/micahgodbolt/pen/GRRvOxg?editors=1010

@micahgodbolt Ah! Nice, this solves my needs. Thanks for the input.

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts (hunk 1 above)
Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts]: this is still a concern

**Comment by micahgodbolt** [comment]
@malmil does this mean you don't need this PR anymore?

**Comment by malmil** [comment]
@micahgodbolt your solution suits my needs. I don't think this feature is necessary.

**Pull request closed without merging**